extensions/amp-story/1.0/amp-story.js

```diff
@@ -1623,6 +1623,9 @@ export class AmpStory extends AMP.BaseElement {
               mutation => mutation.attributeName === 'open')) {
             this.storeService_.dispatch(Action.TOGGLE_SIDEBAR,
                 this.sidebar_.hasAttribute('open'));
+          } if (mutationsList.some(
+              mutation => mutation.attributeName === 'hidden')) {
```

Comment:
This also applied to the earlier code, but I think this case makes it worse: technically, we're looping through all the mutations in the document, right?  So, any element with the `hidden` attribute (whether or not it's actually sidebar-related) could trigger this?

Reply:
Good catch, I just checked and by default it only observes on the actual element, you'd have to pass a boolean `subtree: true` (defaults to false) option to get the callback triggered on any DOM tree mutation